pygfx/utils/enums.py

```diff
@@ -151,6 +151,7 @@ class MarkerShape(Enum):
     spade = None  #: ♠
     club = None  #: ♣
     pin = None  #: 📍
+    custom = None  # Custom shape allowing users to provide their own SDF function
```

Comment:
I think you also need to add `custom` to the docstring in `materials/_points.py` in the setter for marker

Reply:
added. please see new docstrings.